var/spack/repos/builtin/packages/py-urllib3/package.py

```diff
@@ -26,19 +32,27 @@ class PyUrllib3(PythonPackage):
     variant("socks", default=False, description="SOCKS and HTTP proxy support")
     variant("secure", default=False, description="Add SSL/TLS support")
```

Comment:
Disable deprecated `secure` after version 2.0.
```suggestion
    variant("secure", default=False, description="Add SSL/TLS support", when="@:2.0")
```
This will not affect any current packages, as the only package depending on `py-urllib3 +secure` has an explicit upper version limit:
```console
var/spack/repos/builtin/packages/py-azure-cli/package.py:    depends_on("py-urllib3@1.18:1+secure", type=("build", "run"))
```